applications/nrf5340_audio/README.rst

```diff
@@ -0,0 +1,922 @@
+   This happens shortly after programming the development kit and indicates that the gateway device is connected to at least one headset and ready to send data.
+#. Search the list of audio devices listed in the sound settings of your operating system for *nRF5340 USB Audio* (gateway) and select it as the output device.
+#. Connect headphones to the **HEADPHONE** audio jack on both headset devices.
+#. Start audio playback on your PC from any source.
+#. After **LED1** turns solid blue on the headsets and starts blinking blue on the gateway, press the **PLAY/PAUSE** button on a headset.
+   **LED1** starts blinking blue and the audio stream starts on the chosen headset.
+
+   .. note::
+      The audio outputs only to the left channel of the audio jack, even if the given headset is configured as the right headset.
+      This is because of the mono hardware codec chip used on the development kits.
+      If you want to play stereo sound using one development kit, you must connect an external hardware codec chip that supports stereo.
+
+#. Wait for the **LED2** to light up solid green to indicate that the audio synchronization is achieved between the gateway and the chosen headset.
+#. Press the **PLAY/PAUSE** button on the second headset.
+   Both **LED1** and **LED2** have the same behavior as for the first headset.
+#. Press the **VOL+** button on one of the headsets.
+   The playback volume increases for both headsets.
+#. Press the **VOL-** button on the gateway.
+   The playback volume decreases for both headsets.
+#. Press the **PLAY/PAUSE** button on one of the headsets.
+   The playback stops for the given headset and continues on the other one.
+#. Press the **RESET** button on the gateway.
+   The gateway resets and the playback on the unpaused headset stops.
+   After some time, the gateway establishes the connection with both headsets and resumes the playback on the unpaused headset.
+#. Press the **PLAY/PAUSE** button on one of the paused headsets.
+   The playback resumes in sync with the other headset.
+#. Press the **BTN 4** button on the gateway multiple times.
+   For each button press, the audio stream playback is stopped and the gateway sends a test tone to both headsets.
+   These tones can be used as audio cues to check the synchronization of the headsets.
+
+When you finish testing, power off the nRF5340 Audio development kits by switching the power switch from On to Off.
+
+.. _octave_testing_steps_bis:
+
+Testing the BIS mode
+--------------------
+
+Testing the BIS mode is identical to `Testing the default CIS mode`_, except for the following differences:
+
+* You must :ref:`select the BIS mode manually <octave_configuration_select_bis>` before building the application.
+* You can play the audio stream with different audio settings on the receivers.
+  For example, you can decrease or increase the volume separately for each receiver during playback.
+
+.. _octave_porting_guide:
+
+Adapting application for end products
```

Comment:
This is our Porting guide, and we'll keep it at the end for the time being, also because it goes way beyond configuring the current application for testing.